polling_stations/templates/fragments/help_your_council.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+<div class="card">
+    {% blocktrans with council.electoral_services_phone_numbers.0 as council_phone and council.name as council_name and council.electoral_services_email as council_electoral_services_email %}
+        <h3>Your election (might) need you!</h3>
+        <p>
+            A lot of people are needed to make an election happen.
+	    You can contact your council electoral services team to see if they need help.
+        </p>
+        <p>
+            You can <strong>
+		    <a href="{{ council_electoral_services_email }}">email</a>
+	    </strong> {{ council_name }} electoral services on or call them on
```

Comment:
Think there is a rogue `on` here (the first one) that should be removed here? So that it reads "You can email {{ council_name }} electoral services or call them on..."

Reply:
Yes! Thanks :)